addon/-private/system/references.js

```diff
@@ -1,5 +1,5 @@
-import RecordReference from './references/record';
-import BelongsToReference from './references/belongs-to';
-import HasManyReference from './references/has-many';
+import RecordReference from 'ember-data/-private/system/references/record';
```

Comment:
Sure thing. Let's wait until that is resolved, then I will rebase and remove this commit.